static/js/components/CentroidPlotPlugins.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react"; // eslint-disable-line no-unused-vars
+import PropTypes from "prop-types";
+import makeStyles from "@mui/styles/makeStyles";
+import Typography from "@mui/material/Typography"; // eslint-disable-line no-unused-vars
+
+// import * as archiveActions from "../ducks/archive";
+// the import above is needs to be implemented in the codebase where this plugin is used
```

Comment:
I don't understand this comment.

Reply:
argh sorry I typed that like a madlad

Reply:
Done. Do you think it's clear enough now?

Reply:
Can remove is. Otherwise good